packages/storybook/src/styles/theme.css

```diff
@@ -29,3 +29,40 @@
 .section li + li {
   border-top: solid 1px #a7a7a7;
 }
+
+.dt-preview {
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  align-items: center;
+  gap: var(--lux-space-1x, 0.5rem);
+}
+
+/* stylelint-disable selector-class-pattern */
+.dt-preview--spacing,
+.dt-preview--sizing {
+  /* stylelint-enable selector-class-pattern */
+  --dt-border-radius: 0;
+  --dt-background-color: var(--lux-color-background-subdued);
+}
+
+.dt-preview::before {
+  display: inline-block;
+  border-width: var(--dt-border-width, 1px);
+  border-style: var(--dt-border-style, solid);
+  border-color: var(--dt-border-color, var(--lux-color-border-subdued, #a7a7a7));
+  border-radius: var(--dt-border-radius, var(--lux-border-radius-circle, 50%));
+  background-color: var(--dt-background-color, var(--lux-color-background-subdued));
+  aspect-ratio: 1;
+  inline-size: var(--dt-inline-size, 1.25rem);
+  content: "";
+}
+
+/* stylelint-disable-next-line selector-class-pattern */
```

Comment:
Moeten we deze file sowieso niet excluden van stylelint of iig de qua class- en andere patterns?

Reply:
In mijn beleving was dat al zo. Ik zal even kijken wat er aan de hand is

Reply:
Ik heb een `selector-class-pattern` rule toegevoegd voor stylelint